scripts/icm-injection-scripts/inject-icm.sh

```diff
@@ -68,4 +68,7 @@ cat ${icm_output_location}
 
 echo "Appending a COPY command to the Containerfile"
 
+# add new line before and after so it won't be adding at the same line if new line is missing in containerfile
+echo >> "${CONTAINERFILE}"
 echo "COPY $icm_filename $location_in_container" >> "${CONTAINERFILE}"
```

Comment:
Can we modify the test case to not have a newline after the Containerfile too? We just `echo > Containerfile`, can we `echo -n > Containerfile`?

Reply:
https://github.com/konflux-ci/buildah-container/blob/main/scripts/icm-injection-scripts/test-inject-icm.sh#L22